pkg/api/api.go

```diff
@@ -268,6 +269,10 @@ func (a *API) RegisterDistributor(d *distributor.Distributor, pushConfig distrib
 		pushConfig.MaxRecvMsgSize, d.RequestBufferPool, a.sourceIPs, a.cfg.SkipLabelNameValidationHeader,
 		a.cfg.SkipLabelCountValidationHeader, limits, pushConfig.RetryConfig, d.PushWithMiddlewares, d.PushMetrics, a.logger,
 	), true, false, "POST")
+	// TODO(alexg): hidden behind a featureflag or experimental config option?
+	a.RegisterRoute(InfluxPushEndpoint, distributor.InfluxHandler(
```

Comment:
I don't think a feature flag for this is needed. We can just state in the docs (about-versioning.md) that the endpoint is experimental.

Reply:
OK, done.